pkg/client/factory_test.go

```diff
@@ -75,15 +71,8 @@ func TestClientWithPollingConfigManager(t *testing.T) {
 
 func TestClientWithPollingConfigManagerRequester(t *testing.T) {
 
-	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Won't this make it so that the polling config manager starts polling for real? Seems like it could be a lot of noise for the unit tests as we'll start seeing 403s or 404s?

Reply:
addressed